src/Repository/CommitteeMembershipHistoryRepository.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace AppBundle\Repository;
+
+use AppBundle\Entity\Adherent;
+use AppBundle\Entity\Reporting\CommitteeMembershipAction;
+use AppBundle\Entity\Reporting\CommitteeMembershipHistory;
+use Cake\Chronos\Chronos;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Ramsey\Uuid\UuidInterface;
+
+class CommitteeMembershipHistoryRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, CommitteeMembershipHistory::class);
+    }
+
+    public function countAdherentMemberOfAtLeastOneCommitteeManagedBy(Adherent $referent, \DateTimeInterface $until): int
+    {
+        if (!$referent->isReferent()) {
```

Comment:
You can use a function `checkReferent` of this trait https://github.com/EnMarche/en-marche.fr/blob/master/src/Repository/ReferentTrait.php#L9